distsql/request_builder.go

```diff
@@ -138,18 +139,25 @@ func (builder *RequestBuilder) getIsolationLevel(sv *variable.SessionVars) kv.Is
 	return kv.SI
 }
 
+func (builder *RequestBuilder) getKVPriority(sv *variable.SessionVars) int {
+	switch sv.StmtCtx.Priority {
+	case mysql.NoPriority, mysql.DelayedPriority:
```

Comment:
mysql.DelayedPriority equal to kv.PriorityNormal?

Reply:
We don't support this option, so set to default priority.